packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -311,11 +320,16 @@ export function getObjectDefaults<T = any, S extends StrictRJSFSchema = RJSFSche
       experimental_defaultFormStateBehavior?.allOf === 'populateDefaults' && ALL_OF_KEY in schema
         ? retrieveSchema<T, S, F>(validator, schema, rootSchema, formData)
         : schema;
+    const parentConst = retrievedSchema[CONST_KEY];
     const objectDefaults = Object.keys(retrievedSchema.properties || {}).reduce(
       (acc: GenericObjectType, key: string) => {
+        const propertySchema = get(retrievedSchema, [PROPERTIES_KEY, key]);
+        // Check if the parent schema has a const property defined,  then we should always return the computedDefault since it's coming from the const.
```

Comment:
```suggestion
        // Check if the parent schema has a const property defined, then we should always return the computedDefault since it's coming from the const.
```